src/qt/paymentserver.cpp

```diff
@@ -648,7 +648,7 @@ void PaymentServer::fetchPaymentACK(WalletModel* walletModel, const SendCoinsRec
         qWarning() << "PaymentServer::fetchPaymentACK: Error getting refund key, refund_to not set";
     }
 
-    int length = payment.ByteSize();
+    int length = payment.ByteSizeLong();
```

Comment:
This change isn't building... 

> qt/paymentserver.cpp:651:26: error: ‘class payments::Payment’ has no member named ‘ByteSizeLong’; did you mean ‘ByteSize’?
     int length = payment.ByteSizeLong();
                          ^~~~~~~~~~~~
                          ByteSize

Reply:
what version of protobuf do you have?

Reply:
fixed it